src/components/editor/editor-window/tool-bar/emoji-picker/emoji-picker.tsx

```diff
@@ -1,37 +1,48 @@
-import { EmojiData, NimblePicker, Data } from 'emoji-mart'
-import React from 'react'
-import { ShowIf } from '../../../../common/show-if/show-if'
-import emojiData from 'emoji-mart/data/twitter.json'
+import { Data, EmojiData, NimblePicker } from 'emoji-mart'
 import 'emoji-mart/css/emoji-mart.css'
+import emojiData from 'emoji-mart/data/twitter.json'
+import React, { useRef } from 'react'
+import { useClickAway } from 'react-use'
+import { ShowIf } from '../../../../common/show-if/show-if'
 import './emoji-picker.scss'
 import { ForkAwesomeIcons } from './icon-names'
 
 export interface EmojiPickerProps {
   show: boolean
   onEmojiSelected: (emoji: EmojiData) => void
+  onDismiss: () => void
 }
 
-export const EmojiPicker: React.FC<EmojiPickerProps> = ({ show, onEmojiSelected }) => {
+export const EmojiPicker: React.FC<EmojiPickerProps> = ({ show, onEmojiSelected, onDismiss }) => {
+  const pickerRef = useRef(null)
+
+  useClickAway(pickerRef, () => {
+    onDismiss()
+  })
+
   return (
     <ShowIf condition={show}>
-      <NimblePicker
-        data={emojiData as unknown as Data}
-        set={'twitter'}
-        onSelect={onEmojiSelected}
-        theme={'auto'}
-        title=''
-        style={{ top: '96px' }}
-        custom={Object.keys(ForkAwesomeIcons).map((name) => ({
-          name: `fa-${name}`,
-          short_names: [`fa-${name.toLowerCase()}`],
-          text: '',
-          emoticons: [],
-          keywords: ['fork awesome'],
-          imageUrl: 'https://github.githubassets.com/images/icons/emoji/octocat.png',
-          customCategory: 'ForkAwesome'
-        }))
-        }
-      />
+      <div ref={pickerRef}>
+        <NimblePicker
+          ref={pickerRef}
```

Comment:
This looks as if the `div` and the `NimblePicker` both have the same reference object linked?

Reply:
yeah the `NimblePicker` should not have the ref as that did not work and was my initial try

Reply:
fixed